tests/ui/drop/drop_elaboration_with_errors3.rs

```diff
@@ -12,6 +14,7 @@ async fn create_task() -> impl Sized {
 
 async fn documentation() {
     include_str!("nonexistent");
```

Comment:
IMO this should likely use a less fragile error than involving the OS, its localization ("os error 2" could depend on the OS language setting), and normalize-stderr?

```suggestion
    compile_error!("bonjour");
```